src/ts/routing/pages/winnersPage.ts

```diff
@@ -1,6 +1,63 @@
 export function addWinnersPage(): HTMLElement {
   const wrapper = document.createElement('div');
-  wrapper.classList.add('wrapper');
-  wrapper.innerText = 'Winners';
+  wrapper.classList.add('wrapper'); 
+  
+  // Title
+  const winnersTitle = document.createElement('div');
```

Comment:
it's better to create separate function for elements creation to avoid duplicate code  
see pattern 'Fabric'